docs/topics/minishift-install-nexus-setup.adoc

```diff
@@ -0,0 +1,137 @@
+minishift start
+--
++
+[WARNING]
+--
+The procedure described below works with {OpenShiftLocal} 1.0.0-rc2 It has not been tested with the CDK.
+Also make sure that you are using version 3.6.0-alpha.1 or later of the `oc` CLI tool.
+--
+
+* Deploy the {launcher} application to your {OpenShiftLocal}.
+
+* Deploy a Booster application to your {OpenShiftLocal}.
+
+== Configuring your {OpenShiftLocal} to Use Nexus.
+
+. Log in to your {OpenShiftLocal} instance.
++
+[source,bash,subs="attributes+"]
+--
+oc login https://192.168.42.10:8443 -u developer -p developer
+--
++
+. You can reuse the Docker daemon instance used by {OpenShiftLocal} to download the latest versions of the Nexus Docker container image.
++
+[source,bash,subs="attributes+"]
+--
+eval $(minishift docker-env)
+docker pull openshift/jenkins-2-centos7
+docker pull openshiftio/launchpad-jenkins-slave
+docker pull sonatype/nexus
+--
+
+== Setting up the Nexus Application
+
+.  Create a new project to contain the Nexus server. You can also use the _New Project_ button on the Web console to do this.
++
+[source,bash,subs="attributes+"]
+--
+oc new-project NEXUS_PROJECT_NAME
+--
++
+. Create a new application based on the Nexus container image.
++
+[source,bash,subs="attributes+"]
+--
+oc new-app sonatype/nexus
+--
++
+. Expose the service route URL of the Nexus server.
++
+[source,bash,subs="attributes+"]
+--
+oc expose svc/nexus
+--
++
+. Attach a 1GiB persistent storage volume to the Nexus application.
++
+[source,bash,subs="attributes+"]
+--
+oc volumes dc/nexus --add --name 'nexus-volume-1' --type 'pvc' --mount-path '/sonatype-work/' --claim-name 'nexus-pv' --claim-size '1G' --overwrite
+--
++
+.  Navigate to the project containing your Booster application.
++
+[source,bash,subs="attributes+"]
+--
+oc project BOOSTER_PROJECT_NAME
+--
++
+. Assign the `admin` and `edit` permissions to your OpenShift account. You might want to skip this step if you have already assigned these permissions to your account.
++
+[source,bash,subs="attributes+"]
+--
+oc policy add-role-to-user edit system:serviceaccount:BOOSTER_PROJECT_NAME:default
+oc policy add-role-to-user admin system:serviceaccount:BOOSTER_PROJECT_NAME:jenkins
+--
++
+. Obtain the Jenkins server URL and use it to access the Jenkins instance from your browser.
++
+[source,bash,subs="attributes+"]
+--
+oc get routes jenkins
+--
++
+----
+NAME      HOST/PORT                                         PATH      SERVICES   PORT      TERMINATION
+jenkins   jenkins-{app-name}.192.168.42.106.nip.io             jenkins    <all>     edge/Redirect
+----
++
+. Navigate to `+++jenkins-{app-name}.192.168.42.106.nip.io/configure+++`, and log in using your OpenShift credentials to access the Jenkins configuration page.
++
+. In the section titled _Kubernetes Pod Template_,  click on  on _Add Evironmental Variable_ and choose the _Global Environmental Variable_ option form the drop-down menu.
++
+. Define a new environmental with the following properties:
```

Comment:
AFAIK, that both `environment` and `environmental` are used in OpenShift docs, the former being more common of the two forms. There does not seem to be a formal consensus on which form is correct. I'm gonna stick with "environment variable" (lowercase)  for our docs.